patches/components-omnibox-browser-BUILD.gn.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
-index ce702e7dec262a0ff797235efcd5f6f3e5df3531..328ac3af1ff2f74a1e950fe5010478450def3a29 100644
+index ce702e7dec262a0ff797235efcd5f6f3e5df3531..7281b5cf527c5414d28f5dd9901f2e0ab00186e6 100644
 --- a/components/omnibox/browser/BUILD.gn
 +++ b/components/omnibox/browser/BUILD.gn
 @@ -250,6 +250,7 @@ static_library("browser") {
      "//ui/base",
      "//ui/gfx",
    ]
-+  if (brave_chromium_build) {  deps += [ "//brave/components/omnibox/browser" ] }
++  if (brave_chromium_build) {  deps += [ "//brave/components/omnibox/browser" ] sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps }
```

Comment:
you can remove `brave_chromium_build` and you should have two `deps +=` lines here

Reply:
If we move all files of "//brave/components/omnibox/browser" target to sources.gni, it looks like we don't need to add  deps += [ "//brave/components/omnibox/browser" ], don't it?